app/controllers/obituaries/advanced_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Obituaries::AdvancedController < ApplicationController
+
+  def index
+
+    @obituaries_api = ObituarySearch.new.search_advanced_obituaries(params[:name], params[:year])
```

Comment:
just use name and year in call

Reply:
Nice, will it throw an error if only one is entered?

Reply:
just updated it so it will throw an error, push up branch now